apps/cowswap-frontend/src/modules/limitOrders/services/types.ts

```diff
@@ -19,8 +19,7 @@ export interface TradeFlowContext {
   provider: Web3Provider
   allowsOffchainSigning: boolean
   isGnosisSafeWallet: boolean
-  permitInfo: PermitInfo | undefined
-  hasEnoughAllowance: boolean | undefined
+  permitInfo: IsTokenPermittableResult
```

Comment:
Simplified the parameters.
Pass undefined when hasEnoughAllowance is falsy rather than passing that property independently down